autoload/go/lint.vim

```diff
@@ -87,7 +87,7 @@ function! go#lint#Gometa(autosave, ...) abort
 
   let meta_command = join(cmd, " ")
 
-  let out = go#tool#ExecuteInDir(meta_command)
+  let out = go#util#System(meta_command)
```

Comment:
The function comment says:

   " For sync mode (go#tool#ExecuteInDir), always explicitly pass the 5 seconds
   " deadline if there is no other deadline configured. If a deadline is
   " configured, then use it.

But this is no longer accurate.

Could you also update the comment?